doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -396,11 +436,56 @@ Note that this is needed if the model creator returns multiple models.
         }
 
 
+    def custom_validate(config, model, val_iterator, criterion, scheduler=None):
+        """Runs one standard validation pass over the val_iterator.
+
+        Args:
+            config: (dict): A user configuration provided into the Trainer
+                constructor.
+            model: The model(s) as created by the model_creator.
+            train_iterator: An iterator created from the DataLoader which
+                wraps the provided Dataset.
+            criterion: The loss object created by the loss_creator.
+            scheduler (optional): The torch.optim.lr_scheduler object(s)
+                as created by the scheduler_creator.
+
+        Returns:
+            A dict of metrics from the evaluation.
+        """
+        ...
+        return {...}
+
+
     trainer = PyTorchTrainer(
         model_creator,
         data_creator,
         optimizer_creator,
         nn.BCELoss,
         train_function=train,
+        validation_function=custom_validate,
         ...
     )
+
+Feature Requests
+----------------
+
+Have features that you'd really like to see in RaySGD? Feel free to `open an issue <https://github.com/ray-project/ray>`_.
+
+.. _raysgd-pytorch-examples:
+
+PyTorchTrainer Examples
+-----------------------
+
+Here are some examples of using RaySGD for training PyTorch models. If you'd like
+to contribute an example, feel free to create a `pull request here <https://github.com/ray-project/ray/pull/>`_.
```

Comment:
Link broken for me, try this? https://github.com/ray-project/ray/pull/new/master

Reply:
Ah, unfortunately it probably makes more sense to point them to the contributions page - just pointing them to repo for now

```suggestion
to contribute an example, feel free to create a `pull request here <https://github.com/ray-project/ray/>`_.
```